Documentation/gettingstarted/servicemesh/envoy-config.rst

```diff
@@ -0,0 +1,59 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gs_envoy_config:
+
+**************************
+Cilium Envoy Configuration
+**************************
+
+Cilium provides the way to configure envoy proxy via CRDs (e.g. CiliumEnvoyConfig
+and CiliumClusterwideEnvoyConfig).
+
+.. Note::
+
+    This is for advanced scenario only. Cilium Ingress can be used for common use cases.
```

Comment:
Maybe I am missing context, but I don't understand this note. What is the difference between "advanced scenario" and "common use cases"?